siac/hostcmd.go

```diff
@@ -51,6 +52,61 @@ Doing so will override the standard connectivity checks.`,
 	}
 )
 
+// parseSize converts strings of form 10GB to a size in bytes
+// If parsing fails the original value is returned unmodified
+func parseSize(value string) string {
```

Comment:
`value` is an awkward name because it's fully generic. It's the same class of name as `data` or `input` and doesn't give you any indication of what it's supposed to be.

`strSize` is probably better.